modules/service-account-auto-secret-removed.adoc

```diff
@@ -9,13 +9,9 @@
 
 When a service account is created, a service account token secret is automatically generated for it. This service account token secret, along with an automatically generated docker configuration secret, is used to authenticate to the internal {product-title} registry. Do not rely on these automatically generated secrets for your own use; they might be removed in a future {product-title} release.
 
-[NOTE]
+[IMPORTANT]
 ====
-Prior to {product-title} 4.11, a second service account token secret was generated when a service account was created. This service account token secret was used to access the Kubernetes API.
-
-Starting with {product-title} 4.11, this second service account token secret is no longer created. This is because the `LegacyServiceAccountTokenNoAutoGeneration` upstream Kubernetes feature gate was enabled, which stops the automatic generation of secret-based service account tokens to access the Kubernetes API.
-
-After upgrading to {product-version}, any existing service account token secrets are not deleted and continue to function.
+If you disable the integrated {product-registry} by disabling the `ImageRegistry` cluster capability or by setting the Image Registry Operator to `Removed` after installation, secrets are not automatically generated for each service account.
```

Comment:
The deleted text still applies. I would keep the NOTE and add the IMPORTANT section.